views/confirmation/check-answers.pug

```diff
@@ -14,22 +14,24 @@ block content
         h2.breakdown-table__heading #{__(index)}
         dl.breakdown-table-data
           each row in section
-            .breakdown-table__row.breakdown-table__row--answers
-              dt.breakdown-table__row-key #{__(row.text)}
-
-              if (row.text.toLowerCase().includes('address'))
-                dd.breakdown-table__row-value
-                  +address(data)
-              else
-                //- we don't want to translate the name, dollar amounts, or birthday (formatting the birthday is done prior to hitting the template in checkAnswersFormat)
-                if (row.text.toLowerCase().includes('name') || row.data.includes('$') || row.text.toLowerCase().includes('birth'))
-                  dd.breakdown-table__row-value #{row.data}
+            if (data.login.securityQuestion === '/checkAnswers' && !row.text.toLowerCase().includes('income')) || (data.login.securityQuestion !== '/checkAnswers')
```

Comment:
Because of the `/notice` page, we don't always have a securityQuestion. 

So, for example, if I say "no" on the notice page, I see my benefits calculations, which I don't have.

(So if there is no securityQuestion, we should _not_ show the benefits calculations.)